src/data.json

```diff
@@ -34,7 +34,7 @@
             "libraryaddict",
             "Puremin0rez",
             "Intelli",
-            "KennyTV",
+            "kennytv",
```

Comment:
This change is somewhat concerning at first, but considering that this change is necessary for the other changes in this file, I'm okay with this compromise.

Reply:
(￣y▽￣)╭ Ohohoho.....